javascript-if/main.js

```diff
@@ -0,0 +1,94 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  let result;
+  if (number < 5) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isEven(number) {
+  let result;
+  if (number % 2 === 0) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function startsWithJ(string) {
+  let result;
+  if (string.charAt(0) === 'J') {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrink(person) {
+  let result;
+  if (person.age < 21) {
+    result = false;
+  } else {
+    result = true;
+  }
+  return result;
+}
+
+function isOldEnoughToDrive(person) {
+  let result;
+  if (person.age > 16) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age > 21) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function categorizeAcidity(pH) {
+  let result;
+  if (pH > 0 && pH < 7) {
+    result = 'acid';
+  } else if (pH === 7) {
+    result = 'neutral';
+  } else if (pH > 7 && pH < 14) {
+    result = 'base';
+  } else {
+    result = 'invalid pH level';
+  }
+  return result;
+}
```

Comment:
`0` and `14` are valid `pH` levels, so they should return `'acid'` and `'base'` (respectively):

---

![image](https://user-images.githubusercontent.com/16143771/152250417-1c38a56f-a450-4a6f-a59e-175fb59c7bcb.png)